src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -147,29 +147,38 @@
               </a>
             </div>
             <div class="d-flex flex-column ms-4 me-4 mt-4 mb-4">
-              <ng-template #healthChecks>
-                <cd-health-checks *ngIf="healthData?.health?.checks"
-                                  [healthData]="healthData.health.checks"></cd-health-checks>
-              </ng-template>
 
               <div class="d-flex flex-row col-md-3 ms-4">
                 <i *ngIf="healthData.health?.status"
-                   [ngClass]="[healthData.health.status | healthIcon, icons.large2x]"
-                   [ngStyle]="healthData.health.status | healthColor"
-                   [title]="healthData.health.status"></i>
+                    [ngClass]="[healthData.health.status | healthIcon, icons.large2x]"
+                    [ngStyle]="healthData.health.status | healthColor"
+                    [title]="healthData.health.status">
+              </i>
+              <span class="ms-2 mt-n1 lead"
+              *ngIf="!healthData.health?.checks?.length"
+              i18n>Cluster</span>
+              <div cdsPopover
+                  [isOpen]="isOpen"
+                  [dropShadow]="true"
+                  [align]="bottom"
+                  [caret]="true"
+                  [highContrast]="highContrast">
+              <div class="popover-trigger">
                 <a class="ms-2 mt-n1 lead text-primary"
-                   [ngbPopover]="healthChecks"
                    popoverClass="info-card-popover-cluster-status"
-                   [openDelay]="300"
-                   [closeDelay]="500"
-                   triggers="mouseenter"
                    *ngIf="healthData.health?.checks?.length"
+                   (click)="togglePopover()"
```

Comment:
The popover should be closed even if user clicks outside (like on page or other place)
Reference: https://carbondesignsystem.com/components/popover/usage/#interactions

Reply:
Hey @nmunet I have tried doing that.. its from document that it should be the by default nature if we click outside popover, it should be closed but I tried this but its not closing. I think this should be fine we can close popover on click. 
I tried for hover also but user will not be able to scroll over the popover if it will be large.

Reply:
Did you try with (blur) event? if that works?

Reply:
@nmunet I have tried all events blur, focus, click its not closing.

Reply:
I was looking into this, since @nmunet brings a good point here, the popover should close when clicked outside of it.

As far as I've seen: the popover is closed setting `isOpen` property to `false` which removes the `class.cds--popover--open` that keeps it open. So we just need to decide when to set `isOpen` to `false`, in our case when clicked anywhere but the `cds-popover-content`.

Not sure about other approaches, but I tried to capture a click event outside the current popover, and emit it to close the popover. It did work for the most part, but I was not getting the current open `cds-popover-content` properly so even when clicking inside the popover this was closed (not desired).

[screen-capture (12).webm](https://github.com/user-attachments/assets/c832913d-73e9-4f94-8d0e-c5fdedcea092)

In conclusion, I think we should create a wrapper for the `cdsPopover` that emits an event when clicked outside the content of it.

What do you think? @prgoel-code